lib/parse/url.js

```diff
@@ -5,7 +5,6 @@ const isValidUrl = ({ url }) => {
   const isBot = _.includes(['favicon.ico', 'robots.txt'], url)
   const isFileExtension = url.match(/\.[A-z]{2,4}$/)
   const isLegacyEditRoute = url.match(/\/?edit\/?$/)
-  const isDateModifier = url.match(/\/?by\/?$/)
```

Comment:
lets adjust this so that the preceding slash is not optional (remove the first `?`), and do the same with `isLegacyEditRoute`

Reply:
done